x11-plugins/pidgin-funyahoo-plusplus/metadata.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<!-- maintainer-needed -->
```

Comment:
Please don't add new packages with no maintainers.
See https://bugs.gentoo.org/show_bug.cgi?id=590972 and https://projects.gentoo.org/council/meeting-logs/20160814-summary.txt
If you plan to maintain this package via proxy-maint like you've said in the bug, please reflect this fact in metadata.